src/3d/framegraph/qgs3daxisrenderview.h

```diff
@@ -0,0 +1,99 @@
+/***************************************************************************
+  qgs3daxisrenderview.h
+  --------------------------------------
+  Date                 : June 2024
+  Copyright            : (C) 2024 by Benoit De Mezzo
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGS3DAXISRENDERVIEW_H
+#define QGS3DAXISRENDERVIEW_H
+
+#include "qgis_3d.h"
+#include "qgsabstractrenderview.h"
+#include <QObject>
+
+#define SIP_NO_FILE
+
+class QColor;
+class QRect;
+class QSurface;
+
+namespace Qt3DCore
+{
+  class QEntity;
+}
+
+namespace Qt3DRender
+{
+  class QRenderSettings;
+  class QCamera;
+  class QFrameGraphNode;
+  class QLayer;
+  class QViewport;
+  class QSubtreeEnabler;
+  class QRenderTargetSelector;
+}
+
+class Qgs3DMapCanvas;
+
+class QgsFrameGraph;
+class Qgs3DMapSettings;
+
+#define SIP_NO_FILE
+
+/**
+ * \ingroup 3d
+ * \brief 3d axis render view class
+ *
+ * \note Not available in Python bindings
+ * \since QGIS 3.40
+ */
+class _3D_EXPORT Qgs3DAxisRenderView : public QgsAbstractRenderView
+{
+    Q_OBJECT
+  public:
+
+    /**
+     * Constructor for Qgs3DAxisRenderView with the specified \a parent object.
+     */
+    Qgs3DAxisRenderView( QObject *parent, Qgs3DMapCanvas *canvas,
+                         Qt3DRender::QCamera *objectCamera, Qt3DRender::QCamera *labelsCamera, Qgs3DMapSettings *settings );
+
+    //! Returns the viewport associated to this renderview
+    virtual Qt3DRender::QViewport *viewport() override;
+
+    Qt3DRender::QLayer *labelsLayer() { return mLabelsLayer; }
+
+  public slots:
+    //! Updates viewport horizontal \a position
+    void onHorizPositionChanged( Qt::AnchorPoint position );
```

Comment:
Expand abbreviations, eg:

```suggestion
    void onHorizontalPositionChanged( Qt::AnchorPoint position );
```

(elsewhere too)

Reply:
done